libopae/plugins/xfpga/enum.c

```diff
@@ -386,7 +386,7 @@ STATIC fpga_result enum_fme(const char *sysfspath, const char *name,
 	// Discover the socket id from the FME's sysfs entry.
 	resval = sysfs_parse_attribute64(sysfspath, FPGA_SYSFS_SOCKET_ID, &value);
 	if (resval != 0) {
-		return FPGA_NOT_FOUND;
+		//return FPGA_NOT_FOUND;
```

Comment:
Can we consume the fix made by @anandaravuri for this?  Not sure this should be checked in with commented-out error reporting.

Reply:
Yup. This is resolved in the merge conflict.